### ISSUE TITLE
`await-promise` error on mongoose project

### ISSUE THREAD

**Issue by danielpa9708** [saadq/lynt#13]
Hello, I'm trying lynt on a project that uses mongoose, and I got these errors when using await on a `DocumentQuery` type from mongoose that behaves like a promise

```js
const user = await UserModel.findById(id);
```

Is there any way to go around this error

**Comment by danielpa9708** [comment]
With tslint I had something like

```js
        "no-floating-promises": [true, "DocumentQuery", "Query", "MongooseThenable"],
```

**Comment by saadq** [comment]
Unfortunately a lot of the TS errors were added in pretty hastily when I was creating the initial TSLint config, so there are a lot of overly oppressive rules right now that will be removed very soon. They are being kept track of in [this issue](https://github.com/saadq/lynt/issues/12).

I don't think it would be a good idea to try to come up with every case with for Promise-like stuff, so I think it might be just a good idea to remove this rule altogether and add it to that issue. Thoughts?

**Comment by danielpa9708** [comment]
Sounds good for me, maybe we could make a `.lyntrc` file like prettier to allow altering these rules a little

**Comment by saadq** [comment]
Fixed by #16. There is already support for a `.lyntrc` file as noted [here](https://github.com/saadq/lynt#settings), but it is only for the same stuff that the CLI offers like `ignore` and such. I don't really want to add support for configuration of *rules*, because I think that vastly takes away from the point of this project.

The one thing I may consider is adding a `disable` property to the settings which allows you to turn off any of the rules, but I think having something like `enable` wouldn't make sense – the rules that you would want to enable should already exist in `lynt`, and if they don't an issue should be raised about it or there is already a good reason why it shouldn't be in there.

**Comment by saadq** [comment]
Changed my mind, lol

https://github.com/saadq/lynt/issues/18
